tests/actions/orderAction.test.js

```diff
@@ -0,0 +1,26 @@
+import { SET_ORDER_ACTION } from '../../app/js/actions/actionTypes';
+import { setOrderAction } from '../../app/js/actions/orderAction';
+
+describe('Set Order Actions', () => {
+  beforeEach(() => moxios.install());
+  afterEach(() => moxios.uninstall());
+
+  it('should dispatch an order action', async (done) => {
+    const order = {
+      orderNumber: 3
```

Comment:
Could you update the `orderNumber` value to also be of type string as represented in `SearchAndAddOrder.jsx`.
Also, check a similar occurrence in `activeOrderReducer.test.js`

Reply:
okay thanks